src/mpsse.rs

```diff
@@ -4,22 +4,39 @@ use super::{BitMode, DeviceType, FtStatus, FtdiCommon, TimeoutError};
 use std::convert::From;
 use std::time::Duration;
 
+/// MPSSE opcodes.
+///
+/// Exported for use by [`mpsse`] macro. May also be used for manual command array construction.
```

Comment:
```suggestion
/// MPSSE opcodes.
///
/// Exported for use by [`mpsse`] macro. May also be used for manual command array construction.
///
/// Data clocking MPSSE commands are broken out into separate enums for API ergonomics:
/// * [`ClockDataOut`]
/// * [`ClockBitsOut`]
/// * [`ClockDataIn`]
/// * [`ClockBitsIn`]
/// * [`ClockData`]
```

Little more clarification for anyone who finds the data clocking operations missing.